docs/umi/examples.md

```diff
@@ -32,5 +32,10 @@ This is an example of an end-to-end cmd-line-only workflow:
     # Remove poly(A) tails and concatemer
     $ isoseq3 refine flt.bam primers.fasta fltnc.bam --require-polya
 
+    # Correct Barcodes
+    $ isoseq3 correct --barcodes include.txt fltnc.bam corrected.bam
+
+    $samtools sort -t CB corrected.bam -o corrected.sorted.bam
```

Comment:
Move this to after the "#deduplicate" header and right before "isoseq3 groupdedup" command